javascript/javascript1/week3/noteTakingApp.js

```diff
@@ -0,0 +1,37 @@
+//Note taking app
+
+const notes = [];
+
+function saveNote(content, id){
+   return notes.push({content: content, id: id});
+}
+
+saveNote('Pick up groceries', 1);
+saveNote('Do Laundry', 2);
+saveNote('Clean', 3);
+
+console.log(notes);
+
+function getNote(id){
+  for (let i = 0; i < notes.length; i++){
+    if(notes[i].id === id){
+      return notes[i];
+    }
+  }
+    if(!id){
```

Comment:
What if it's there, but it's not a number?

Reply:
I have changed it to check for NAN